_sass/_popup.scss

```diff
@@ -6,26 +6,30 @@
 }
 
 input[type="checkbox"] + label {
```

Comment:
Should this style decleration be scoped within .popup?

Reply:
Better be